### ISSUE TITLE
Description of options Use Delay Detection and Use Latency Correction

### ISSUE THREAD

**Issue by redchar** [Soreepeong/XivAlexander#65]
I can't find a description of these options anywhere for users, just that Use Delay Detection doesn't work well for some people and shouldn't be used with VPNs.

What do these options actually do? Are they just supposed to further reduce delay for some users, or are they to more accurately emulate natural low latency and avoid unintentional cheating?

Are you just supposed to try enabling/disabling them and see what works best for you?

For example, I play on ~200 ping (no VPN) and find that with both options enabled I can double weave fine, but disabling Use Delay Detection feels more responsive. Is disabling this option giving me an unfair advantage if I don't "need" it?

**Comment by jjolano** [comment]
> accurately emulate natural low latency and avoid unintentional cheating

Precisely this. Every client request (action) has some processing time at the server-side, so the program with the "Use Delay Detection" option enabled will factor in the time it takes for the server to respond minus your ping.

For the "Use Latency Correction" option, it will consider an additional factor based on the fluctuation of your ping to the server to prevent some actions being randomly faster than others. Additionally, it will also attempt to reduce the impact of larger spikes in ping.

With both options disabled, it will use a static value of 675ms (animation lock ms + 75ms "processing time") between actions. This may feel more responsive, especially if the calculated processing time is higher than 75ms, but I feel this can theoretically be easier to detect.

**Comment by VXLSS** [comment]
This is good information, thank you. I am in a weird position where I have to use a VPN AND this xiva and I cannot play without them both. After reading this I tried enabling the options and disable my VPN and it was unplayable, my combos stopped procing and I could double weave but sometimes oGCDs would get ghosted. I turned on VPN and almost the same scenario, so now I am playing as I was with VPN on and Delay detection off. I hope it doesn't make me easily detectable.
